### PR TITLE
update: canto chain rpc endpoints & explorer endpoints

### PR DESCRIPTION
**Pull request by jessepinkman9900**

### CODE REVIEW

**Review by Bjonesca30 — APPROVED** [review]

**Comment by jessepinkman9900** [comment]
Thanks @Bjonesca30! Can you merge this as well?

File: _data/chains/eip155-7700.json [hunk 1]
```diff
@@ -13,14 +13,19 @@
   "chainId": 7700,
   "networkId": 7700,
   "explorers": [
+    {
+      "name": "Canto EVM Explorer (Blockscout)",
+      "url": "https://tuber.build",
+      "standard": "none"
+    },
```
Comment (ligi) [inline _data/chains/eip155-7700.json:20]: please add this on the end

File: _data/chains/eip155-7700.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Canto",
   "chain": "Canto",
-  "rpc": ["https://canto.slingshot.finance"],
+  "rpc": ["https://canto.neobase.one", "https://canto.slingshot.finance"],
```
Comment (ligi) [inline _data/chains/eip155-7700.json:4]: please add the new RPC as last entry

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by jessepinkman9900** [comment]
have resolved the issues. please review @ligi 
thanks!

**Review by ligi — APPROVED** [review]

**Pull request merged**